### DISCUSSION TITLE
how to view tasks without a project

### DISCUSSION THREAD

**Discussion by esquires** [nvim-neorg/neorg#212] [Q&A] [answered]
Greetings. neorg is a great project!

I am trying to figure out how to access random tasks that may not be in the inbox. Right now they show up in the projects list (from `:Neorg gtd vews` then hitting `p`) as

```
*Projects*

- /2 tasks don't have a project assigned/

| /Projects with no Area Of Focus/

```

When I hit enter (or tab) on the line with `2 tasks don't...` it doesn't change anything. I was thinking it might adjust the view so that there would be a list of tasks (that can then be jumped to). Is there a way to view those tasks? Thanks!

An example that generates this view is in `actions.norg` with

```
@document.meta
    title: actions
    description: 
    author: esquires3
    categories: 
    created: 2021-11-29
    version: 0.0.8
@end

- [ ] first task not in a project

* My Project
  - [x] first task in project
  - [x] second task in project

* A Section That Is Not a Project

Here is some information about stuff.

===
getting out of the section

- [ ] second task not in project
```

**Comment by danymat** [comment]
> Greetings. neorg is a great project!

Thanks bro !

> When I hit enter (or tab) on the line with 2 tasks don't... it doesn't change anything. I was thinking it might adjust the view so that there would be a list of tasks (that can then be jumped to). Is there a way to view those tasks? Thanks!

I didn't add this functionality at the moment, you can press tab on tasks and projects only.
It should be quite easy to implement this feature I think

**Comment by danymat** [comment, marked as answer]
Hello, just added the capabilities in this commit: 5100593b

Can you check it out ?

**Reply by esquires** [reply]
Looks great! After hitting "tab" on the line with "/2 tasks..." it expands perfectly. 

However, when I hit "enter" on one of those expanded lines it just takes me to the file with the task. Can it take me to the line with the task?

**Reply by danymat** [reply]
> Can it take me to the line with the task?

It is somehow more complex than it sounds, and I would need a proper issue tracking this. If you want this feature, feel free to open an issue !

**Reply by esquires** [reply]
No problem. Will do.
